cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/runnable/DemoteBrokerRunnable.java

```diff
@@ -95,7 +96,7 @@ public DemoteBrokerRunnable(KafkaCruiseControl kafkaCruiseControl,
     super(kafkaCruiseControl, future, parameters.dryRun(), null,
           parameters.stopOngoingExecution(), null, true, null,
           parameters.allowCapacityEstimation(), parameters.excludeRecentlyDemotedBrokers(), false,
-          uuid, parameters::reason, true);
+          uuid, parameters::reason, true, false);
```

Comment:
I noticed that we didn't include fastMode parameter in DemoteBrokerParameters. Is the reason being we seldom observed potential timeouts?
If we decide to expose this parameter, is setting demote fastMode default to true a good idea? The reason being the same as described in #1575. My only concern is that all other fastMode default would be false, so accepting my suggestion might lead to some confusion.
```suggestion
          uuid, parameters::reason, true, true);
```
BTW, I agree that all other fastMode defaults should be false.

Reply:
Demote broker is a special case that uses a hardcoded goal (i.e. `PreferredLeaderElectionGoal`), which is expected to run fast. To verify my assertion, I ran a demote operation in a cluster with `270+` brokers. The optimization of `PreferredLeaderElectionGoal` in this cluster took `1402 ms`. Hence, given its special case, at least for the time being, I feel we can avoid exposing the parameter for this endpoint.